README.md

```diff
@@ -80,6 +80,16 @@ extensions:
 
 https://github.com/phylum-dev/cli/tree/main/extensions
 
+### musl support
+
+Following the release of the extensions feature, as of version 3.8.0, Phylum CLI
+has a dependency on `glibc` and thus can no longer support `musl` compilation targets.
+
+This means that `phylum` won't be executable in environments such as Alpine Linux.
+
+If your use case requires a lightweight Docker base image, consider using
+[Debian slim](https://hub.docker.com/_/debian).
+
```

Comment:
I don't like the idea of saying that we "can no longer support" musl. Users can still build from source to get binaries that are dynamically-linked against musl. And some truly advanced user may even find a way to build a statically-linked version.

The change is the binaries that we provide. Here's a possible way to write this section:

```suggestion
### libc version

As of version 3.8.0, the provided Linux binaries of the Phylum CLI depend on
`glibc`. We no longer provide binaries that are statically compiled with `musl`
libc

This means the provided binaries won't be executable in environments such as
Alpine Linux.

If your use case requires a lightweight Docker base image, consider using
[Debian slim](https://hub.docker.com/_/debian) or [building from
source](https://docs.phylum.io/docs/alternate_install)
```